### DISCUSSION TITLE
"Low Disk Space" warning preventing me from accessing codespace

### DISCUSSION THREAD

**Discussion by mger1608** [community/community#120755] [GitHub Education]
### Hi everyone,

I am attempting to work on a project that is stored in a GitHub Codespace. I have been working on this in my free time for the past month without any GitHub Codespace related issues. Today, when trying to unzip a file via a line of my code I received a 'no disk space' error message and the following warning popped up in my VS Code IDE.

![Screenshot 2024-04-22 123814](https://github.com/community/community/assets/59756802/10c3a813-d5f4-48cf-ab39-2b2c8d63fe6c)

I'm not really sure what this means but after trying to troubleshoot on my own I have concluded that my GitHub Codespace is maxxed out from a storage/usage standpoint. However, when I view the usage report for my codespace online, it appears that I have plenty of storage and usage available. 

![Screenshot 2024-04-22 123806](https://github.com/community/community/assets/59756802/14a414db-1577-4f46-9b74-ec779b206582)

I tried to simply disconnect and reconnect to the codespace via the Remote Explorer tab in VS Code. Upon doing so, I can no longer even open a file from my codespace, much less try to run anything. I am stuck with a loading bar at the top and a blank editor attempting to open 3 different files.

![Screenshot 2024-04-22 124739](https://github.com/community/community/assets/59756802/c76f5e1b-7f44-4ccf-b9af-9e50396508e4)

Is this a known issue? Is there something obvious I may be doing incorrectly? Appreciate any feedback.

**Comment by samus-aran** [comment]
Hi @mger1608 👋 

Is this specifically Codespaces or are you using Classroom?

**Reply by mger1608** [reply]
@samus-aran This is specifically Codespaces being run on VS Code

**Reply by samus-aran** [reply]
Hi @mger1608 

Students are able to use Codespaces whilst within Classroom without affecting their personal Codespaces limits. I assume that is what has happened. I would suggest you reach out to [accounts & billing support](https://support.github.com/contact/account) to gain clarification as yes that can be confusing to see on the dashboard.

**Comment by huywashere** [comment]
Hi
